src/main/java/org/reactome/release/qa/check/MissingEditorialAttributeCheck.java

```diff
@@ -0,0 +1,99 @@
+import org.reactome.release.qa.common.QAReport;
+
+/**
+ * This QA check was requested by Lisa.
+ * Description:
+ * "Weekly QA and release QA: Report new events in
+ * slice (those where stableID released attribute
+ * is != True) and edited, reviewed, or authored
+ * attribute is null."
+ * @author sshorser
+ *
+ */
+@SliceQACheck
+public class MissingEditorialAttributeCheck extends AbstractQACheck {
+
+	private static final String YES = "YES";
+	private static final String NO = "NO";
+	private static Set<String> reportedEntities = new HashSet<>();
+
+	@Override
+	public QAReport executeQACheck() throws Exception
+	{
+		GKSchemaClass eventClass = (GKSchemaClass) this.dba.getSchema().getClassByName(ReactomeJavaConstants.Event);
+		QAReport report = new QAReport();
+		report.setColumnHeaders("DB_ID", "Name", "Edited?", "Authored?", "Reviewed?", "Created");
+		Collection<GKSchemaClass> classesToCheck = (Collection<GKSchemaClass>) eventClass.getSubClasses();
+		// make sure Event is in there with its subclasses.
+		classesToCheck = new ArrayList<>(classesToCheck);
+		classesToCheck.add(eventClass);
+		for (GKSchemaClass schemaClass : classesToCheck)
+		{
+			Collection<GKInstance> eventInstances = (Collection<GKInstance>) this.dba.fetchInstancesByClass(schemaClass);
+			for (GKInstance eventInstance : eventInstances)
+			{
+				String[] line = getReportLine(eventInstance);
+				if (line.length > 0)
+				{
+					report.addLine(line);
+				}
+			}
+		}
+		return report;
+	}
+
+	/**
+	 * Returns an array of Strings that will be added as a line to the report - IFF the eventInstance meets the criteria:
+	 * StableIdentifier is NOT released; Any of: authored, edited, reviewed are null.
+	 * @param eventInstance
+	 * @return
+	 * @throws InvalidAttributeException
+	 * @throws Exception
+	 */
+	private String[] getReportLine(GKInstance eventInstance) throws InvalidAttributeException, Exception
+	{
+		ArrayList<String> line = new ArrayList<>();
+		GKInstance stableIdentifier = (GKInstance) eventInstance.getAttributeValue(ReactomeJavaConstants.stableIdentifier);
+		Boolean released = (Boolean) stableIdentifier.getAttributeValue(ReactomeJavaConstants.released);
+		String dbId = eventInstance.getDBID().toString();
+		// Check that the instance hasn't already been reported. It _could_ happen, since we're traversing all subclasses of Event.
+		if ((released == null || !released.booleanValue()) && !reportedEntities.contains(dbId))
+		{
+			// We have found an unreleased Event! Now we need to check the editorial attributes.
+			GKInstance edited = (GKInstance) eventInstance.getAttributeValue(ReactomeJavaConstants.edited);
+			GKInstance authored = (GKInstance) eventInstance.getAttributeValue(ReactomeJavaConstants.authored);
+			GKInstance reviewed = (GKInstance) eventInstance.getAttributeValue(ReactomeJavaConstants.reviewed);
+
+			boolean editedIsNull = edited == null;
+			boolean authoredIsNull = authored == null;
+			boolean reviewedIsNull = reviewed == null;
```

Comment:
This can be tricky to read at a glance due to an assignment and comparison on the same line.  Could parentheses be added around the comparison for clarity?  

For example: `boolean editedIsNull = (edited == null);`

Reply:
I am not a fan of this change. It is an uncommon use of parentheses that I find makes the code weirder looking and harder to read.